deployment/entity-service/templates/configmap.yaml

```diff
@@ -5,8 +5,6 @@ metadata:
   labels:
     {{- include "es.release_labels" . | indent 4 }}
 data:
-  DEBUG: {{ required "workers.debug is required." .Values.workers.debug | quote }}
```

Comment:
what was that for? Why don't we need it anymore?

Reply:
The `debug` flag was removed ages ago from the python code and settings. Has just taken a while to remove all traces - now we configure logging in a different config map.